lib/main.coffee

```diff
@@ -49,8 +38,8 @@ module.exports =
         filePath = textEditor.getPath()
         command = @executablePath
         ruleset = @rulesets
-        if @projectRules
-          rulesetPath = helpers.find(filePath, 'ruleset.xml')
+        if /^[a-z0-9]+\.xml$/gi.test(@rulesets)
```

Comment:
How about this? 
![skjermbilde 2016-05-09 kl 22 55 11](https://cloud.githubusercontent.com/assets/81981/15127906/200d2864-1639-11e6-931a-cbbfc80b5869.png)